### PR TITLE
PNI: Detail page WIP

### PR DESCRIPTION
**Pull request by gvn**

### CODE REVIEW

**Comment by gvn** [comment]
https://foundation-mofostaging-pr-1846.herokuapp.com/privacynotincluded/

**Comment by kristinashu** [comment]
Design QA notes are in [this doc](https://docs.google.com/document/d/14wTXrEmYmQFWYlmkwFXJtHJZPiseXTgAokJq7_BHx28/edit?usp=sharing). I will put these into a separate follow up ticket.

**Comment by gvn** [comment]
@alanmoo in the interest of keeping code reviews on the smaller side I think we should land this chunk. Have a look see and let me know if there are any glaring issues bearing in mind that it's very WIP.

**Comment by kristinashu** [comment]
I've put QA list in a ticket https://github.com/mozilla/foundation.mozilla.org/issues/1860

File: network-api/networkapi/buyersguide/models.py [hunk 1]
```diff
@@ -144,6 +233,42 @@ class Product(models.Model):
         blank="True",
     )
 
+    update1 = models.ForeignKey(
```
Comment (alanmoo) [inline network-api/networkapi/buyersguide/models.py]: This also feels like a weird way to implement this. @gideonthomas do you happen to know of a DRYer, simple way to do this?

[the remark this answers is not in the document]
Comment (gvn) [reply network-api/networkapi/buyersguide/models.py]: Do you happen to know how to do that?

**Pull request merged**